core/task/class.go

```diff
@@ -51,6 +53,19 @@ type info struct {
 	Constraints []constraint.Constraint `yaml:"constraints"`
 }
 
+type taskClassIdentifier struct {
+	repo repos.Repo
+	Name string
+}
+
+func (tcID taskClassIdentifier) String() string {
+	if tcID.repo.Revision != "" {
+		return fmt.Sprintf("%vtasks/%v@%v", tcID.repo.GetIdentifier(), tcID.Name, tcID.repo.Revision)
```

Comment:
`%s` should work here if all substitution inputs are strings.

Reply:
Done